### ISSUE TITLE
URL request error, can dynamic update and re-request？

### ISSUE THREAD

**Issue by songlingling0012** [spring-cloud/spring-cloud-openfeign#833]
If the specified url parameter causes a request failure, can you replace the url parameter value to send the request again？

Kind of like FallBackFactory, but based on the global configuration of FeignClient, and able to send the server a second request instead of returning an exception directly

**Comment by OlgaMaciaszek** [comment]
No, at this point, you would have to implement the logic in the fallback class.
